packages/config/src/chrome-render-loader.js

```diff
@@ -0,0 +1,5 @@
+function chromeRenderLoader(source) {
+    return `if(!window.isChrome2){${JSON.stringify(source)}}`;
```

Comment:
I would change this to something more insights specific `insights.chrome.isChrome2`

Reply:
That can be would be a problem. The flag should be set in the chrome HTML template to ensure that it's there even if the chrome was not loaded yet.

If we put that into the chrome init function it may be too late.

Reply:
NVM, i see that the chrome init is not overriding the existing chrome object on the window. We can change it.